controllers/account.controller.js

```diff
@@ -30,6 +30,31 @@ async function getUserByEmail(req, res) {
     }
 }
 
+/**
+ * @async
+ * @function getUserById
+ * @param req
```

Comment:
What are the types for these objects?

Reply:
Done

Reply:
I'll also fix this issue on the other PR

Reply:
I'll update req and res for all the controllers.